test/alias-record/schema_test.go

```diff
@@ -13,22 +14,22 @@ const fixtureJson = `
 [
   {
     "OtherField": {
-      "StringField": "789", 
-      "BoolField": true, 
+      "StringField": "789",
+      "BoolField": true,
       "BytesField": "VGhpcyBpcyBhIHRlc3Qgc3RyaW5n"
     }
   },
   {
     "OtherField": {
-      "StringField": "abcdghejw", 
-      "BoolField": true, 
+      "StringField": "abcdghejw",
+      "BoolField": true,
       "BytesField": "VGhpcyBpcyBhIHRlc3Qgc3RyaW5n"
     }
   },
   {
     "OtherField": {
-      "StringField": "jdnwjkendwedddedee", 
-      "BoolField": true, 
```

Comment:
Removing trailing whitespace is helpful, but it adds a lot of noise to this PR. For things like renaming variables and removing whitespace it would be easier for me to review a separate PR that doesn't also add new features.

Reply:
> I don't think we need to use the SOE package for the integration tests. It should be sufficient to write a couple unit tests in the soe package for serializing and deserializing headers.

I would advice to add soe to at least one test to cover soe integration with the serialization/deserialization though.